pkg/operation/cloudbotanist/awsbotanist/controlplane.go

```diff
@@ -127,6 +128,12 @@ func (b *AWSBotanist) GenerateCSIConfig() (map[string]interface{}, error) {
 	return nil, nil
 }
 
+// MetadataServiceAddress returns AWS's MetadataService address.
+func (b *AWSBotanist) MetadataServiceAddress() (*net.IPNet, error) {
```

Comment:
Do we even need this function, you are statically passing the IPNet output yourself as a string, what's the value, if I provide `169.254.169.254/32` I get back `169.254.169.254/32`, MetadataService IPs are always `/32` maybe it would have been useful in different cases where you are not always using a `/32`.